src/main/groovy/org/jetbrains/intellij/tasks/DownloadRobotRegistryPluginTask.groovy

```diff
@@ -0,0 +1,25 @@
+package org.jetbrains.intellij.tasks
+
+import org.gradle.api.internal.ConventionTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.TaskAction
+import org.jetbrains.intellij.dependency.PluginDependency
+import org.jetbrains.intellij.dependency.PluginDependencyImpl
+
+class DownloadRobotRegistryPluginTask extends ConventionTask {
+    @Input
+    String robotServerVersion
+
+    @TaskAction
+    void download() {
+        def sourceUrl = "https://bintray.com/jetbrains/intellij-third-party-dependencies/download_file?file_path=org%2Fjetbrains%2Ftest%2Frobot-server-plugin%2F${robotServerVersion}%2Frobot-server-plugin-${robotServerVersion}.zip"
```

Comment:
No need to download maven dependency manually. Gradle can do this for you. See instrument task, it downloads compiler from maven repo

Reply:
fixed